Goldtree/Goldtree/Program.cs

```diff
@@ -86,204 +83,228 @@ public static void Initialize()
             Console.Write(" XorTroll");
             Console.WriteLine();
             Console.ForegroundColor = ConsoleColor.Gray;
-            Warn.Log("Make sure to open Goldtree after having launched Goldleaf on your console. Waiting for connection...");
+            Warn.Log(Messages.EnsureConnection);
         }
 
-        [STAThread]
-        public static void Main(string[] Args)
+        private void InitializeUSBConnection()
         {
-            usbnsp:
-            Console.Clear();
-            Initialize();
+            
+            Log.Log(Messages.ConnectingToUSB);
             try
             {
                 var pat = new KLST_PATTERN_MATCH { DeviceID = @"USB\VID_057E&PID_3000" };
                 var lst = new LstK(0, ref pat);
                 lst.MoveNext(out var dinfo);
-                USB = new UsbK(dinfo);
+                this.USB = new UsbK(dinfo);
             }
-            catch
+            catch (libusbK.DriverLoadException)
```

Comment:
If you pull out the Program logic and this becomes a reusable GoldLeaf class, it should insulate the user from internal libusbK exceptions and perhaps throw its own exceptions (e.g. `GoldLeafRuntimeException`). Alternatively, it can handle these exceptions and Initialize could return a Boolean `false`.